src/test/java/no/unit/nva/model/IssnValidatorTest.java

```diff
@@ -0,0 +1,35 @@
+package no.unit.nva.model;
+
+import no.unit.nva.model.exceptions.InvalidIssnException;
+import no.unit.nva.model.validator.IssnValidator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class IssnValidatorTest {
+
+    @DisplayName("The valid ISSNs pass")
+    @Test
+    void issnCreatesIssnWithValidInput() {
+        List<String> validIssns = List.of("1476-4687", "0363-6941", "1945-662X", "0025-5629");
```

Comment:
You can use ParameterizedTest: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests

Reply:
Experimental feature?